examples/07_joining_tables_with_FuzzyJoin.py

```diff
@@ -0,0 +1,161 @@
+"""
+Joining tables with fuzzyjoin
+================================
+
+In this example, we show how to join tables with the :function:`fuzzyjoin` function. 
+We also demonstrate why this method is the most easy and appropriate tool for handling 
+the joining of tables for users that want to improve their machine learning models quickly.
+We go through some of the rich options that allow for manipulation when joining tables,
+such as taking into account the precision of the match.
+
+To do so, we will predict the happiness score of a country from
+the 2022 [World Happiness Report](https://worldhappiness.report/).
+We will also use data provided from the [World Bank open data platform](https://data.worldbank.org/)
+in order to create a satisfying prediction model.
+
+"""
+
+###############################################################################
+# Data Importing and preprocessing
+# --------------------------------
+#
+# We import the happiness score data first:
+import pandas as pd
+
+df = pd.read_csv('https://raw.githubusercontent.com/jovan-stojanovic/data/main/Happiness_report_2022.csv', thousands=',')
```

Comment:
I don't think that we should be hosting data on a person's repository. But we can host it in the dirty-cat organization

Reply:
Would this work https://github.com/dirty-cat/datasets/pull/5 ? When you review it, I can add the final link to the example.